lib/core/assertion.js

```diff
@@ -214,6 +214,11 @@ module.exports = new (function() {
         }).replace(/:/g, '').replace(/\s/g, '-').replace(/-\(.+?\)/, '');
         var fileName = path.resolve(path.join(client.options.screenshots.path, 'FAIL_' + message + theBrowser + dateStamp + '.png'));
         client.api.saveScreenshot(fileName, function(result) {
```

Comment:
Is the "result" from the saveScreenshot function being used in the callback function?